pgdrive/envs/pgdrive_env.py

```diff
@@ -176,14 +181,17 @@ def lazy_init(self):
         # init vehicle
         v_config = self.config["vehicle_config"]
         self.vehicle = BaseVehicle(self.pg_world, v_config)
+        self.vehicles = {i: BaseVehicle(self.pg_world, v_config) for i in range(self.config["num_player"])}
```

Comment:
I would suggest to leave a `def initialize_vehicle` function in original PGDrive, so that in your new class, say `MultiAgentPGDriveEnv`, you can simply overwrite the `initialize_vehicle` function.